src/components/Header/Header.jsx

```diff
@@ -5,21 +5,21 @@ import logo from '../../images/ui/Logo.svg';
 import headerStyles from './Header.module.scss';
 
 export default function Header() {
-	// @TODO ко всем картинкам подключить общий стиль чтобы они были блочными
+  // @TODO ко всем картинкам подключить общий стиль чтобы они были блочными
 
-	return (
-		<header className={headerStyles.header}>
-			<Link to="/">
-				<img className={headerStyles.headerImage} src={logo} alt="Логотип" />
-			</Link>
-			<div className={headerStyles.headerUserContainer}>
-				<img src="#" alt="фото профиля" />
-				<button
-					className={headerStyles.button}
-					type="button"
-					aria-label="выйти из профиля"
-				/>
-			</div>
-		</header>
-	);
+  return (
+    <header className={headerStyles.header}>
+      <Link to="/">
+        <img className={headerStyles.headerImage} src={logo} alt="Логотип" />
+      </Link>
+      <div className={headerStyles.headerUserContainer}>
+        <img src="#" alt="фото профиля" />
+        <button
+          className={headerStyles.button}
+          type="button"
+          aria-label="выйти из профиля"
```

Comment:
```suggestion
          aria-label="Выйти из профиля"
```
относитесь к alt и aria-label так же, как и просто к надписям